src/java/com/zimbra/cs/taglib/tag/LoginTag.java

```diff
@@ -284,11 +284,15 @@ public static void setTrustedCookie(HttpServletResponse response, String trusted
         }
     }
 
-    public static boolean isCaptchaValid(String secretKey, String response) {
+    public static boolean isCaptchaValid(String captchaId, String captchaInput) {
         try {
-            String url = "https://www.google.com/recaptcha/api/siteverify?"
-                    + "secret=" + URLEncoder.encode(secretKey, "UTF-8")
-                    + "&response=" + URLEncoder.encode(response, "UTF-8");
+		    if (captchaId == null || captchaInput == null) {
+			  result = false;
+			 }
+			
+            String url = "http://web02.anahar.dev.opal.synacor.com:8666/verifyCaptcha?"
```

Comment:
(Same as https://github.com/Zimbra/zm-web-client/pull/267/files)
That doesn't seem like a good default URL to use or hardcode.
If it is removed in another ticket/PR, please reply to the comment.

Reply:
We'll remove this and will make URL configurable via LDAP attribute. This require a discussion with Russ either we'll do this in same ticket or in next ticket.